# lidar parameters.
lidar_name: "HDL-32"
reverse_flag: false

# line extraction parameters.
v_min_line_points: 4
v_seed_line_points: 3
v_least_thresh: 0.06
v_min_line_length: 0.06
v_predict_distance: 0.12
v_pts_missing_tolerance: 1
v_max_pts_gap: 1

h_min_line_points: 5
h_seed_line_points: 3
h_least_thresh: 0.06
h_min_line_length: 0.04
h_predict_distance: 0.12
h_pts_missing_tolerance: 5
h_max_pts_gap: 1.5

# line clustering parameters.
line_plane_distThreshold: 0.078


